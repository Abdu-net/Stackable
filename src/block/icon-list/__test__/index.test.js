import { edit, save } from '../index'

describe( 'Icon List', () => {
	test( 'block edit matches snapshots', () => {
		const wrapper = edit( {
			isSelected: false,
			attributes: {
				icon: 'check',
				iconShape: 'circle',
				iconColor: '#000000',
				iconSize: 20,
				columns: 1,
				text: 'Some Text',
				gap: 16,
			},

		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( 'block save matches snapshots', () => {
		const wrapper = save( {
			isSelected: false,
			attributes: {
				icon: 'check',
				iconShape: 'circle',
				iconColor: '#000000',
				iconSize: 20,
				columns: 1,
				text: 'Some Text',
				gap: 16,
			},
		} )

		expect( wrapper ).toMatchSnapshot()
	} )
} )
