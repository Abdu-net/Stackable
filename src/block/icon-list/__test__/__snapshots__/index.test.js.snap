// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon List block saved block HTML shouldn't change 1`] = `
<ul
  className="wp-block-ugb-icon-list ugb-icon-list ugb-icon--icon-cross ugb-icon--columns-0 className"
  style={
    Object {
      "--gap": "8px",
      "--icon": "url('data:image/svg+xml;base64,PGRpdj48L2Rpdj4=')",
      "--icon-size": "10px",
    }
  }
>
  <div
    dangerouslySetInnerHTML={
      Object {
        "__html": "<li>text</li>",
      }
    }
  />
</ul>
`;

exports[`Icon List block saved default block HTML shouldn't change 1`] = `
<ul
  className="wp-block-ugb-icon-list ugb-icon-list ugb-icon--icon-check ugb-icon--columns-1"
  style={
    Object {
      "--gap": "16px",
      "--icon": "url('data:image/svg+xml;base64,PGRpdj48L2Rpdj4=')",
      "--icon-size": "20px",
    }
  }
>
  <div
    dangerouslySetInnerHTML={
      Object {
        "__html": "",
      }
    }
  />
</ul>
`;
